.github/workflows/claude-pr-review.yml

```diff
@@ -1,12 +1,20 @@
 name: Claude PR Code Review
 
 on:
+  issue_comment:
+    types: [created]
   pull_request_review_comment:
     types: [created]
+  pull_request_review:
```

Comment:
As far as I understand, this part is not needed, and the `issue_comment` should be the fix that allows it to run on PR comments.